platform/android/src/style/lights.cpp

```diff
@@ -0,0 +1,22 @@
+#include "lights.hpp"
```

Comment:
Maybe make this part of light.{hpp,cpp}? Unless we're expecting different light classes, the naming is a bit odd.